Documentation/applications/games/brickmatch/index.rst

```diff
@@ -3,7 +3,8 @@
 =========================
 
 Brickmatch is a kind puzzle game like a mix between tetris and Candy
-Crush. It is a 6x6 matrix with blocks (cells) with different colors.
+Crush. It is matrix (e.g 6x6, 8x8) with blocks (cells) with different
```

Comment:
@eren-terzioglu Actually it is a fixed size 6x6 game: https://www.hackster.io/nuttxspace/brick-match-a-natural-interface-game-that-all-makerlab-need-ffbfba different sizes are not supported